ts/Series/PackedBubble/PackedBubbleSeries.ts

```diff
@@ -1151,6 +1167,26 @@ class PackedBubbleSeries extends BubbleSeries implements Highcharts.DragNodesSer
         return arr;
     }
 
+    public pointAttribs(
+        point?: PackedBubblePoint,
+        state?: StatesOptionsKey
+    ): SVGAttributes {
+        let options = this.options,
+            hasParentMarker = point && point.isParentNode,
+            markerOptions = hasParentMarker && options.layoutAlgorithm && options.layoutAlgorithm.parentNodeOptions ?
+                options.layoutAlgorithm.parentNodeOptions.marker :
+                options.marker,
+            fillOpacity = (markerOptions as any).fillOpacity,
+            attr = Series.prototype.pointAttribs.call(this, point, state);
+
+        if (fillOpacity !== 1) {
+            attr.fill = color(attr.fill as any)
+                .setOpacity(fillOpacity)
+                .get('rgba');
+        }
```

Comment:
Instead of converting the fill to `rgba`, it is probably better to add `attr['fill-opacity']`.